.changeset/eighty-starfishes-listen.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/firestore": patch
+"@firebase/util": minor
+---
+
+Fix encoding of CSI keys in Safari
```

Comment:
It's better to write this from the dev's perspective / what will appear in the release notes. e.g:

Fix Safari/WebKit cache issues when client-side indexing is used.